src/Basque/Mews.Fiscalizations.Basque/Model/InvoiceState.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Mews.Fiscalizations.Basque.Model
+{
+    public enum InvoiceState
+    {
+        Received = 00,
+        Refused = 01
```

Comment:
You know that 00 is still just 0 and 01 is still just 1, right? :D

Reply:
Yup, im just using whatever is in the technical documentation :D 

Since things are not in english, you can simply copy paste the value and search in the doc :D 

and i believe its easier to parse this way.